conf/messages

```diff
@@ -181,6 +181,18 @@ healthReason.whenHealthIssueHappened.error.required = The date you became unable
 healthReason.whenHealthIssueHappened.error.invalid = The date you became unable to manage VAT must be a real date
 healthReason.whenHealthIssueHappened.error.notInFuture = The date you became unable to manage VAT must be today or in the past
 
+# When did the hospital stay begin page
+# ----------------------------------------------------------
+healthReason.whenDidHospitalStayBegin.title = When did the hospital stay begin?
+healthReason.whenDidHospitalStayBegin.heading = When did the hospital stay begin?
+healthReason.whenDidHospitalStayBegin.hintText = Tell us the date when the hospital stay began. For example, 12 3 2018
```

Comment:
`Tell us the date when the hospital stay began.` part of the hintText is not needed. Just `For example, 12 3 2018` according to the prototype.